dotnet/example_code_legacy/README.md

```diff
@@ -0,0 +1,42 @@
+# README for AWS SDK for .NET Documentation Examples - Legacy code examples
+
+This is a collection of legacy code examples for the AWS SDK for .NET public documentation.  Legacy code examples may use older versions of the .NET framework, and they may not contain unit tests.  
+
+You can find new and updated code examples in the folder example_code.  Legacy code examples may be updated and moved from the example_code_legacy to the example_code folder. 
+
+
+# Prerequisites
+
+To build and run the code examples in this collection, you'll need:
+
+- The AWS SDK for .NET. For more information, see the [AWS SDK for .NET
+Developer Guide](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/welcome.html).
+
+- AWS credentials, either configured in a local AWS credentials file, or by setting the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables.  For more information, see the [AWS Tools and SDKs Shared Configuration and Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/overview.html).
+
+
+# Building and running code examples
+
+For each code example, follow the instructions in the readme file for the example.  In general, follow these steps:
+
+- Open the solution file for the code example in Visual Studio.
+- Compile and run the solution.
+
+
+## Additional information
+
+- As an AWS best practice, grant all code least privilege, or only the permissions required to perform a task. For more information, see [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) in the AWS Identity and Access Management User Guide.
```

Comment:
*AWS Identity and Access Management User Guide*

Reply:
Updated!